src/Marker.tsx

```diff
@@ -14,6 +15,7 @@ export interface MarkerProps extends H.map.Marker.Options, H.geo.IPoint {
   bitmap?: string;
   data?: any;
   draggable?: boolean;
+  children?: React.ReactElement<any>;
```

Comment:
There is a bug report regarding ReactElement<any> https://github.com/DefinitelyTyped/DefinitelyTyped/issues/27805
Please use a generic node instead as we might run into a problem later on.
It's not important anyways as type safety is removed in prod.